earlystopping-medianstop/rockcraft.yaml

```diff
@@ -1,65 +1,68 @@
+# Based on https://github.com/kubeflow/katib/tree/master/cmd/earlystopping/medianstop/v1beta1
 name: earlystopping-medianstop
 base: ubuntu:22.04
-version: '1-beta1'
+version: 'v0.15.0_22.04_1'
 summary: Early stopping algorithm for katib.
 description: |
     The median stopping rule stops a pending trial X at step S if the trial’s best 
     objective value by step S is worse than the median value of the running averages 
     of all completed trials' objectives reported up to step S.
 license: GPL-3.0
-platforms:
-    amd64: {}
-    # original dockerfile supports also ppc64le and arm64;
-    # at the moment we only support amd64.
-
+run-user: _daemon_
 services:
-  katib-earlystopping-medianstop:
-    command: /usr/bin/katib-earlystopping-medianstop
+  earlystopping-medianstop:
     override: replace
+    summary: "earlystopping-medianstop service"
     startup: enabled
+    command: "python3 /opt/katib/cmd/earlystopping/medianstop/v1beta1/main.py"
```

Comment:
You can try to use working-dir like this

```
services:
  foo:
    override: replace
    command: main.py
    working-dir: /opt/katib/cmd/earlystopping/medianstop/v1beta1/
```

Reply:
Good suggestion. Will updated.